### ISSUE TITLE
Create a deployable bootstrap layer for Lamb-D.

### ISSUE THREAD

**Issue by truedat101** [IoTone/Lamb-D#7]
Need to figure out what this needs to be.  If it's just a library, ok.  Does it need some glue code, or is it actually just an app?

Looks like this would be starting from a lambda image, and share this out after building it up with the right libraries.

**Comment by ghost91-** [comment]
Not completely sure if I understand correctly, but preferably, Lamb-D should simply be a library I can call, i.e. my own code should look like this:
```d
import lambd;

void main()
{
    runHandler((JSONValue evt, LambdaContext ctx) => {/* ... */});
}
```

In order to provide correct setup and packaging, Lamb-D could provide a [custom dub init type](https://dlang.org/changelog/2.086.0.html#custom-dub-init-type).

**Comment by truedat101** [comment]
Agree.  I spent too much time reading the new Lambda layer/runtime docs. 

But in terms of bootstrap what I think I was getting at was that there needs to be something that can be deployed and shared easily.  Reading on this: https://docs.aws.amazon.com/lambda/latest/dg/runtimes-walkthrough.html

they get into moving from packaging everything together (function + runtime) and separate the two.  So a third party (any D user) could import the runtime and build a layer that references the runtime as a dependency, if I understand this approach correctly.  That was my intent with this issue is to explore the optimal way forward on that.  

Will look into custom dub init type.
